docs/source/tutorials/create_a_fips_updates_pro_cloud_image.md

```diff
@@ -0,0 +1,55 @@
+# Create a customized Cloud Ubuntu Pro image with FIPS Updates
+
+## Step 1: Launch an Ubuntu Pro instance on your cloud
+
+See the following links for up to date information for each supported cloud:
+* https://ubuntu.com/aws/pro
+* https://ubuntu.com/azure/pro
+* https://ubuntu.com/gcp/pro
+
+## Step 2: Enable FIPS Updates
+
+First wait for the standard Ubuntu Pro services to be set up.
+
+```bash
+sudo ua status --wait
```

Comment:
Might be a nitpick, but since we will be transitioning to `pro`, could we replace it here? The integration test also uses `pro`.

Reply:
We won't change it in documentation until after the release of the new name.